### PR TITLE
pytorch-aarch64: add neoverse-v1 & n2 build options

### PR DESCRIPTION
**Pull request by snadampal**

### CODE REVIEW

File: docker/pytorch-aarch64/cpu_info.sh [hunk 1]
```diff
@@ -89,14 +95,30 @@ function set_target {
...
+      tune="neoverse-v1"
+      arch="armv8.4-a"
+      blas_cpu="NEOVERSEV1"
+      blas_ncores=64
+      acl_arch="armv8.2-a"
+    ;;
+    neoverse-n2 )
+      cpu="neoverse-n2"
+      tune="neoverse-n2"
+      arch="armv8.5-a"
```
Comment (nSircombe) [inline docker/pytorch-aarch64/cpu_info.sh:117]: Same request as on https://github.com/ARM-software/Tool-Solutions/pull/109, I think a comment about v9 support would be a good idea.

**Review by nSircombe — COMMENTED** [review]
Looks good, just one suggestion.

File: docker/pytorch-aarch64/cpu_info.sh (hunk 1 above)
Comment (snadampal) [reply docker/pytorch-aarch64/cpu_info.sh:119]: addressed.

**Comment by snadampal** [comment]
> I think this needs a note here to make it clear. You could also apply the .patch from the corresponding OpenBLAS PR over the OpenBLAS release, if it will apply cleanly.

Will leave a note. I didn't take the .patch route because 
1. the patch is already merged into openBLAS
2. patch doesn't apply on openblas v0.3.9 (which is the version currently, without my other PR)
3. as you know, its only a stop gap solution till we get openblas v.3.20. I thought this is the light weight change than maintaining the bulky patch.

**Review by nSircombe — APPROVED** [review]
Looks good to me.
Thanks for your PR; and for your patience!

**Comment by nSircombe** [comment]
@snadampal - this looks good to me, as with the TF PR, I've made a minor re-wording to one of the comments, but nothing significant. 

Thanks for taking the time to make these improvements.

**Pull request merged**